battery-status/battery-promise-iframe.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Battery Test: each Navigator object has a battery promise for BatteryManager</title>
```

Comment:
I guess you mean test files' titles. The tests' names have been specified individually.